nistats/regression.py

```diff
@@ -330,3 +329,17 @@ def SSE(self):
     def MSE(self):
         """ Mean square (error) """
         return self.SSE / self.df_resid
+
+
+def concatenate_regression_results(results):
+    """Concatenate results from a parallelized regression"""
+    theta = np.concatenate([result.theta for result in results], axis=1)
+    Y = np.concatenate([result.Y for result in results], axis=1)
+    model = results[0].model
+    wY = np.concatenate([result.wY for result in results], axis=1)
+    wresid = np.concatenate([result.wresid for result in results], axis=1)
+    dispersion = np.concatenate([result.dispersion for result in results],
+                                axis=0)
+    cov = results[0].cov
+    return RegressionResults(theta, Y, model, wY, wresid,
+                             dispersion=dispersion, cov=cov)
```

Comment:
I am a bit worried about np.concatenate, that can lead to unexpected behavior.
If all this is only used for OLS regression, I do not think that it is worth the effort.